packaging/aznfs/DEBIAN/postinst

```diff
@@ -38,7 +39,23 @@ if [ -f /opt/microsoft/aznfs/randbytes ]; then
         chattr -f +i /opt/microsoft/aznfs/data/randbytes
 fi
 
-# Start aznfswatchdog service.
-systemctl daemon-reload
-systemctl enable aznfswatchdog
-systemctl start aznfswatchdog
\ No newline at end of file
+
+# Check if the config file exists; if not, create it.
+if [ ! -f /opt/microsoft/aznfs/data/config ]; then
+        # Create the config file and set default AUTO_UPDATE_AZNFS=true inside it.
+        echo "AUTO_UPDATE_AZNFS=true" > /opt/microsoft/aznfs/data/config
+
+        # Set the permissions for the config file.
+        chmod 0644 /opt/microsoft/aznfs/data/config
+fi
+
+# If it's an auto upgrade triggered by aznfswatchdog, don't restart watchdog.
+if [ ! -f /tmp/.update_in_progress_from_watchdog.flag ]; then
```

Comment:
I see you have used falg_file var for this file name in other files. Since it is used here as well multiple times, you can use the same var name here as well to be consistent.

Reply:
@palashvij-msft this one is still remaining.